### PR TITLE
os/board/rtl8721csm: Update BT Coex table

### PR DESCRIPTION
**Pull request by ziyik**
- Update BT Wifi Coex table for rtl8721csm
- Fix BT Coex performance drop when AP Beacon interval is not 100ms 
- Included fix for Security Vulnerabilities in FragAttacks

### CODE REVIEW

File: os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/common/src/hci_process.c [hunk 1]
```diff
@@ -22,7 +22,7 @@ extern void hci_normal_start(void);
 extern bool bt_check_iqk(void);
 extern bool hci_start_iqk(void);
 extern uint8_t  hci_tp_lgc_efuse[0x20];
-extern uint8_t  hci_tp_phy_efuse[16];
+extern uint8_t  hci_tp_phy_efuse[19];
```
Comment (jeongchanKim) [inline os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/common/src/hci_process.c:25]: How about defining the value? Then it is easy to maintain.

Comment (ziyik) [reply os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/common/src/hci_process.c:25]: We tend to retain the same, is that okay?
But if insist, we can change it.

Comment (sunghan-chang) [reply os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/common/src/hci_process.c:25]: It's chipset-vendor code. You can do as you want (It's up to you). This is just suggestion for readability, but we have no plan to check and to modify this code.

Comment (ziyik) [reply os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/common/src/hci_process.c:25]: Thank you for the suggestion.
However, we will remain as it is.

File: os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/hci/bt_mp_patch.c [hunk 2]
```diff
@@ -1,675 +1,475 @@
+
 /**
```
Comment (thapav) [inline os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/hci/bt_mp_patch.c:2]: Kindly revert file permission>
build/tools/amebad/gnu_utility/km0_image2_all.bin

Comment (ziyik) [reply os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/hci/bt_mp_patch.c:2]: Sorry, my mistake.
Reverted.

**Review by an4967 — APPROVED** [review]
This works well.

**Pull request merged**